modules/odc-monitoring-your-project-metrics.adoc

```diff
@@ -11,21 +11,34 @@ After you create applications in your project and deploy them, you can use the *
 .Procedure
 
 . On the left navigation panel of the *Developer* perspective, click *Observe* to see the *Dashboard*, *Metrics*, *Alerts*, and *Events* for your project.
-
-* Use the *Dashboard* tab to see graphs depicting the CPU, memory, and bandwidth consumption and network related information, such as the rate of transmitted and received packets and the rate of dropped packets.
 +
-.Monitoring dashboard
-image::odc_project_dashboard.png[]
+. Optional: Use the *Dashboard* tab to see graphs depicting the following application metrics:
++
+* CPU usage 
+* Memory usage
+* Bandwidth consumption
+* Network-related information such as the rate of transmitted and received packets and the rate of dropped packets.
++
```

Comment:
Remove the + in L21

Reply:
@smunje1 Refer to this tip in asciiDoc quick reference: https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#enclose-in-open-block

This will be helpful for future references to too to fix the lists.